site/content/help/feature/multiplayer/index.md

```diff
@@ -0,0 +1,118 @@
+
+Steps:
+
+1. If you do not have a *yuzu* installation yet, install it with the installer from our [official website](https://yuzu-emu.org).
+1. Navigate to the folder you installed *yuzu* to, and go into `yuzu-windows-msvc` or `yuzu-windows-msvc-early-access`.
+    * If you cannot recall the installation folder, right clicking on a shortcut and clicking `Open File Location` will directly bring you into the `yuzu-windows-msvc` or `yuzu-windows-msvc-early-access` folder.
+    * Make sure the folder you are in contains the `yuzu-room` executable.
+1. Create a new text file and copy the following script inside, **fill in the arguments** (they will be explained later) and then use `File > Save As...` to save it to `host.bat`. You can delete the original text file after that.
+1. You can now double click on the batch script named `host` to start hosting.
+
+```
+yuzu-room ^
+  --room-name "<INSERT ROOM NAME HERE>" ^
+  --room-description "<INSERT ROOM DESCRIPTION HERE>" ^
+  --preferred-game "<INSERT GAME TITLE HERE>" ^
+  --preferred-game-id "<INSERT TITLE ID HERE>" ^
+  --port 5000 ^
+  --max_members 4 ^
+  --token "<INSERT yuzu ACCOUNT TOKEN HERE>" ^
+  --enable-yuzu-mods ^
+  --web-api-url https://api.yuzu-emu.org/ ^
+  --ban-list-file "<INSERT BAN LIST FILE PATH HERE>"
+pause
+```
+
+![example](windows_hosting_1.png)
+
+## Command Line Arguments
+Both the *Docker* image and the `yuzu-room` executable takes a number of command line arguments.
+Command line arguments should be specified in the format of `--<name> <value>`, for example `--port 5000`.
+
+By default, rooms will be hosted Public. If you'd like to host a private one instead, just omit one of the fields required only for public rooms.
+
+|Name|Required|Description
+|----|--------|-----------
+|`--room-name`|Yes|Name of your room.
+|`--room-description`|No|An optional long description of your room.
+|`--preferred-game`|Yes|The name of the preferred game of your room.
+|`--preferred-game-id`|No|The Title ID of the preferred game of your room, in hex. You can find the `Title ID` with the game list of yuzu.
+|`--port`|Yes|Port number to host your room on (0-65535). You should use a port that is not already used by other applications (typically a high enough port will work). Though unrecommended due to bandwidth reasons, it is possible to host multiple rooms on a single server provided the ports are distinct. **Remember to configure `Port Forwarding` for the ports!**
+|`--max-members`|Yes|Maximum number of members for your room (2-16). *Note that rooms that are too large may result in slowdowns or even disconnections in certain games.*
+|`--password`|No|An optional password to limit access to your room.
+|`--token`|Public Rooms|Your *yuzu* Community user token. Do not know your token yet? Get it [here](https://yuzu-emu.org/wiki/yuzu-web-service/). **Please note, the token format has changed as of 11/1/2019 and there is now no need for a `--username` field.**
```

Comment:
```suggestion
|`--token`|Public Rooms|Your *yuzu* Community user token. Do not know your token yet? Get it [here](https://yuzu-emu.org/wiki/yuzu-web-service/).
```
Deleted reference to username in Wiki article, removing the need to have this outdated disclaimer